libse/SubtitleFormats/PhoenixSubtitle.cs

```diff
@@ -0,0 +1,137 @@
+        {
+            get
+            {
+                return "Phoenix Subtitle";
+            }
+        }
+
+        public override bool IsMine(List<string> lines, string fileName)
+        {
+            if (fileName != null && !fileName.EndsWith(".pjs", StringComparison.OrdinalIgnoreCase))
+            {
+                return false;
+            }
+            var sub = new Subtitle();
+            LoadSubtitle(sub, lines, fileName);
+            return sub.Paragraphs.Count > _errorCount;
+        }
+
+        public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string fileName)
+        {
+            subtitle.Paragraphs.Clear();
+            _errorCount = 0;
+            var paragraph = new Paragraph();
+            for (int i = 0; i < lines.Count; i++)
+            {
+                string line = lines[i].Trim();
+                Match match = null;
+                if (line.Length >= 4)
+                {
+                    match = RegexTimeCodes.Match(line);
+                }
+                if (match?.Success == true)
+                {
+                    try
+                    {
+                        // Decode times.
+                        paragraph.StartTime = DecodeTimeCode(match.Groups[1].Value);
+                        paragraph.EndTime = DecodeTimeCode(match.Groups[2].Value);
+
+                        // Decode text.
+                        line = line.Substring(match.Value.Length).Trim();
+                        // Read quote (") if there are more than one.
+                        if (line.Length > 2 && line[0] == '"' && line[1] == '"')
+                        {
+                            line = "\"" + line.Trim(TrimChars) + "\"";
+                        }
+                        else
+                        {
+                            line = line.Trim(TrimChars);
+                        }
+
+                        paragraph.Text = string.Join(Environment.NewLine, line.Split('|'));
+                        subtitle.Paragraphs.Add(paragraph);
+                        paragraph = new Paragraph();
+                    }
+                    catch
+                    {
+                        _errorCount++;
+                    }
+                }
+                else
+                {
+                    _errorCount++;
+                }
+            }
+            subtitle.Renumber();
+        }
+
+        private static TimeCode DecodeTimeCode(string encodedTime)
+        {
+            // Format is timed in tenths of seconds. 2447
+            int time;
+            if (int.TryParse(encodedTime, out time))
+            {
+                return new TimeCode(time * .1 * TimeCode.BaseUnit);
+            }
+            return new TimeCode(0);
+        }
+
+        private static string EncodeTimeCode(TimeCode tc)
+        {
+            int tm = (int)Math.Round(tc.TotalSeconds);
+            return Convert.ToString(tm * 10);
+        }
```

Comment:
Is rounding to whole seconds intended? Or should it be rounded to tenths of seconds?
```
    return Convert.ToString((int)Math.Round(tc.TotalMilliseconds / 100.0));
```